### DISCUSSION TITLE
Vote for unconsolable as RisingLight member

### DISCUSSION THREAD

**Discussion by skyzh** [risinglightdb/risinglight#770] [Member]
During the last few months @unconsolable has made great contribution for the RisingLight's storage part. Their contribution includes:

* better encoding for storage format https://github.com/risinglightdb/risinglight/pull/704 https://github.com/risinglightdb/risinglight/pull/709 https://github.com/risinglightdb/risinglight/pull/713 https://github.com/risinglightdb/risinglight/pull/724
* internal executor for execution layer https://github.com/risinglightdb/risinglight/pull/755
* storage bootstrap process https://github.com/risinglightdb/risinglight/pull/760

They are also active in reviewing PRs and participating in discussions in the community. With their solid contribution to the project, I propose to promote @unconsolable to a RisingLight member.

If you are a member of the RisingLight organization, you may vote for this proposal with 👍 and 👎. According to [GOVERNANCE](https://github.com/risinglightdb/risinglight/blob/main/GOVERNANCE.md), a minimum requirement of 2 votes are required for accepting this proposal. Thanks!

**Comment by skyzh** [comment]
With 2 votes from @skyzh and @wangrunji0408, we are accepting this vote. Welcome to the RisingLight community! @unconsolable

**Reply by unconsolable** [reply]
Thanks everyone!
